index.js

```diff
@@ -43,22 +48,40 @@ function start () {
     }
 
     argv.inputPath = path.isAbsolute(argv.input) ? argv.input : path.join(process.cwd(), argv.input)
-    const results = require(argv.inputPath)
-    const report = buildReport(results)
-    writeReport(report, argv.outputPath, (err) => {
-      if (err) console.err('Error writting report: ', err)
-      else console.log('Report written to: ', argv.outputPath)
-    })
-  } else {
-    const concatStream = concat((res) => {
-      const results = JSON.parse(res.toString())
-      const report = buildReport(results)
+    var compare = []
+    if (argv.compare) {
+      argv.compare.forEach(function (val) {
+        val = path.isAbsolute(val) ? val : path.join(process.cwd(), val)
+        fs.access(val, fs.F_OK, function (err) {
+          if (!err) {
+            var json = require(val)
+            compare.push(json)
+          } else {
+            console.log('Can\'t access ' + val)
+          }
+        })
+      })
+      compare.sort()
```

Comment:
`compare = sort(compare)`. compare.sort needs the sort function you defined below